src/utils/AuthService.js

```diff
@@ -83,7 +83,7 @@ export default class AuthService extends EventEmitter {
     this.storage.setItem('id_token', idToken);
 
     Cookies.set('authorization', `Bearer ${idToken}`, {
-      domain: `.${window.location.hostname}`
```

Comment:
ah then in that case we need to treat prod and local environments differently--the reason the cookie wasn't showing up was because it was set for ".localhost" so you couldn't see it at localhost.